TradingBot/Controllers/ManualTestsController.cs

```diff
@@ -87,6 +89,34 @@ public async Task<ActionResult> ExecuteTaskAsync()
         await _taskExecutor.ExecuteAsync(HttpContext.RequestAborted);
         return NoContent();
     }
+
+    [HttpGet]
+    [Route("strategy-results")]
+    [ProducesResponseType(typeof(IReadOnlyList<TradingAction>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<ActionResult<IReadOnlyList<TradingAction>>> GetStrategyResultsAsync()
+    {
+        var result = await _strategy.GetTradingActionsAsync(HttpContext.RequestAborted);
+
+        if (result is null) return NotFound();
+
+        return Ok(result);
+    }
+
+    [HttpGet]
+    [Route("trading-cycle-test")]
+    [ProducesResponseType(typeof(IReadOnlyList<DailyTradingData>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<ActionResult<IReadOnlyList<DailyTradingData>>> ExecuteCycleActionsAsync()
+    {
+        var now = DateTimeOffset.UtcNow;
+        await _executor.ExecuteTradingActionsAsync(HttpContext.RequestAborted);
+        var resultAfterCycleExecution = await _actionQuery.GetTradingActionsAsync(now, DateTimeOffset.UtcNow, HttpContext.RequestAborted);
+
+        if (resultAfterCycleExecution is null) return NotFound();
+
+        return Ok(resultAfterCycleExecution);
+    }
 }
```

Comment:
We don't need this after #25. `trading-tasks` endpoint can be used to execute whole 'trading cycle'.